docs/features.md

```diff
@@ -0,0 +1,9 @@
+### Features
+
+| Features | Device-LocalPV Version | K8S Version |
+| :--- | :--- | :--- |
+| Volume Provisioning | 0.1+ | 1.14+ |
+| Volume Deprovisioning | 0.1+ | 1.14+ |
+| Volume Stats | 0.1+ | 1.15+ |
+| Raw Block Volume | 0.1+ | 1.14+ |
+| Storage Capacity Tracking | 0.1+ | 1.19+ |
```

Comment:
we need 1.20+, as it is in alpha in 1.19+.

Reply:
Done.